test/merger-test.lua

```diff
@@ -765,4 +765,28 @@ for _, input_type in ipairs({'buffer', 'table', 'tuple'}) do
     end
 end
 
+test:test('hotreload', function(test)
```

Comment:
I would catch the 'declared_ctype' [test](https://github.com/tarantool/tuple-keydef/blob/109c91361b9d84a70e6dafd79afadd60eca136ed/test/hot_reload.test.lua#L34-L59) as well. However I can do it as follow up to don't tire you.

Reply:
Yes, moved test hotreload suite from tuple-keydef module into separate file.